app/Filament/Admin/Resources/NodeResource/Widgets/NodeStorageChart.php

```diff
@@ -39,20 +39,18 @@ class NodeStorageChart extends ChartWidget
 
     protected function getData(): array
     {
-        /** @var Node $node */
-        $node = $this->record;
+        $total = Number::format(($this->node->statistics()['disk_total']) / 1024 / 1024 / 1024, maxPrecision: 2, locale: auth()->user()->language);
+        $used = Number::format(($this->node->statistics()['disk_used']) / 1024 / 1024 / 1024, maxPrecision: 2, locale: auth()->user()->language);
 
-        $total = ($node->statistics()['disk_total'] ?? 0) / 1024 / 1024 / 1024;
-        $used = ($node->statistics()['disk_used'] ?? 0) / 1024 / 1024 / 1024;
         $unused = $total - $used;
```

Comment:
This can throw a `A non-numeric value encountered` error when using a locale that has `,` as decimal separator.
That's because `$total` and `$used` use `Number::format`.